standard/expressions.md

```diff
@@ -4562,7 +4565,20 @@ The conditional operator is right-associative, meaning that operations are group
 
 The first operand of the `?:` operator shall be an expression that can be implicitly converted to `bool`, or an expression of a type that implements `operator true`. If neither of these requirements is satisfied, a compile-time error occurs.
 
-The second and third operands, `x` and `y`, of the `?:` operator control the type of the conditional expression. If both `x` and `y` are *default_literal*s ([§12.8.20](expressions.md#12820-default-value-expressions)), a compile-time error occurs.
+If `ref` is present:
```

Comment:
@jskeet to make a suggestion for a note on how this relates to `= ref`

Reply:
@jskeet 

I made a note here in the next commit. That way, we don't lose this discussion. We can update the text of the note via a suggestion or PR after I merge these into a single feature branch.